index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/mobile.css">
+    <link rel="stylesheet" href="styles/desktop.css">
+    <title>Garvin Chimone</title>
+</head>
+<body>
+    <nav>
+        <div class="title"><h2>My Logo</h2></div>
+        <div class="burger-menu">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+    </nav>
+    
+    <section class="headline">
+        <div class="headline-intro">
+            <h1>Hey There. I'm Garvin.</h1>
+
+            <h2>I'm a Software Developer</h2>
+    
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't hesitate to contact me.
+            </p>
+        </div>
+
+        <ul>            
+            <li><img src="images/Vectorgithub.svg" alt="github-icon"></li>
+            <li><img src="images/Grouplinkedin.svg" alt="linkedin-icon"></li>
+            <li><img src="images/Vectorpeace.svg" alt="peace-icon"></li>
+            <li><img src="images/Vectortwitter.svg" alt="twitter-icon"></li>
+            <li><img src="images/Vectorem.svg" alt="M-icon"></li>
+        </ul>
```

Comment:
- [ ] Kindly consider wrapping each `icon` element with an `a` tag. This will provide a better user experience by allowing users to easily connect with you through your social media handles. By associating each `icon` with an `a` tag, it conveys the intention of linking to your social media profiles. This way, users can click on the icons and be directed to the corresponding social media platforms.